src/App.jsx

```diff
@@ -4,14 +4,18 @@ import TodoList from './TodoList.jsx';
 import AddTodoForm from './AddTodoForm.jsx';
 
 const App = () => {
-  const [newTodo, setNewTodo] = useState("");
+  let [todoList, setTodoList] = useState([]);
+
+  let addTodo = (newTodo) => {
+    setTodoList([...todoList, newTodo]);
+  };
 
   return (
     <>
       <h1>Todo List</h1>
-      <AddTodoForm onAddTodo={setNewTodo} />
-      <p>{newTodo}</p>
-      <TodoList />
+      <AddTodoForm onAddTodo={addTodo} />
+      <p></p>
```

Comment:
remove unused elements

Reply:
I have removed this unused paragraph from my code.